src/main/java/com/evilcorp/bloom/model/Product.java

```diff
@@ -8,16 +8,97 @@ public class Product {
   @Id
   private String id;
 
-  private String name;
+  private String productName;
   private String description;
-  private String category;
-  private String brand;
+  private Integer category;
```

Comment:
Will this store the id for the category? if yes, it would be better to be `categoryId`